Yoast/wordpress-seo
# SVG icons: add accessible attribute and clean-up.

## Summary

This PR can be summarized in the following changelog entry:

* Improve SVG images accessibility

## Relevant technical choices:

As done in https://github.com/Yoast/yoast-components/pull/280 and https://github.com/Yoast/my-yoast/pull/1549 
- add attributes `role="img" aria-hidden="true" focusable="false"`
- remove other tags and attributes that have no use in the rendered SVGs, for example: `<title>Link_in_icon</title>`, `id="Laag_5"`, `data-name="&lt;Samengesteld pad&gt;"`, `data-name="Laag 5"`
- changes are applied to all the SVGs I was able to find, even the ones used as backgrounds so they're all standardized to the same format and to make sure there are no SVGs with pointless tags/attributes around

## Test instructions

- check the SVG images still look OK
- note: I understand checking _all_ the SVGs manually is tedious, please notice the changes here just add or remove attributes that don't affect the rendering so I guess checking tehre are no syntax errors is sufficient

Fixes #8140 
Fixes #8685
